auslib/test/test_db.py

```diff
@@ -2103,6 +2118,128 @@ def testGetRulesMatchingQuerySystemCapabilitiesNoSubstringMatch(self):
         rules = self._stripNullColumns(rules)
         self.assertEquals(rules, [])
 
+    def testGetRulesMatchingQuerySystemCapabilitiesOrs(self):
+        match_1 = self.paths.getRulesMatchingQuery(
+            dict(product='a', channel='a', buildTarget='h', buildID='', locale='',
+                 osVersion='', distribution='', distVersion='', headerArchitecture='',
+                 force=False, queryVersion=6, systemCapabilities='GenuineIntel',
+                 version='5.0'),
+            fallbackChannel=''
+        )
+        match_2 = self.paths.getRulesMatchingQuery(
+            dict(product='a', channel='a', buildTarget='h', buildID='', locale='',
+                 osVersion='', distribution='', distVersion='', headerArchitecture='',
+                 force=False, queryVersion=6, systemCapabilities='SSE2', version='5.0'),
+            fallbackChannel=''
+        )
+        no_rules = self.paths.getRulesMatchingQuery(
+            dict(product='a', channel='a', buildTarget='h', buildID='', locale='',
+                 osVersion='', distribution='', distVersion='', headerArchitecture='',
+                 force=False, queryVersion=6, systemCapabilities='SSE3', version='5.0'),
+            fallbackChannel=''
```

Comment:
Awesome catch - I added these cases and changed the code to do exact matches instead of substring matches so that a rule containing "SSE" won't match an incoming rule like "SSE3".